react/features/file-sharing/components/web/FileSharing.tsx

```diff
@@ -281,63 +300,79 @@ const FileSharing = () => {
             }
             { files.size > 0 && (
                 <div className = { classes.fileList }>
-                    { Array.from(files.entries()).sort((a, b) => a[1].fileName.localeCompare(b[1].fileName)).map(([ fileId, file ]) => (
-                        <div
-                            className = { classes.fileItem }
-                            key = { fileId }
-                            title = { file.fileName }>
-                            { (file.progress ?? 100) === 100 && (
-                                <>
-                                    <div className = { classes.fileIconContainer }>
-                                        <Icon
-                                            color = { BaseTheme.palette.icon01 }
-                                            size = { 64 }
-                                            src = { getFileIcon(file.fileType) } />
-                                    </div>
-                                    <div className = { classes.fileItemDetails }>
-                                        <div className = { classes.fileName }>
-                                            { file.fileName }
-                                        </div>
-                                        <div className = { classes.fileSize }>
-                                            { formatFileSize(file.fileSize) }
+                    {
+                        Array.from(files.entries()).sort((a, b) => a[1].fileName.localeCompare(b[1].fileName)).map(([ fileId, file ]) => {
```

Comment:
Please move the whole `Array.from` part to the top where the files` are extracted so we don't need to do it here inline.